api/config/config_local.json

```diff
@@ -125,5 +125,19 @@
     "dataset": "reporting_local",
     "maxRowsPerInsert": 200,
     "uploadMethod": "STREAMING"
+  },
+  "accessTiers": {
+    "registered": {
+      "tierLabel": "Registered Tier",
+      "servicePerimeterName": "accessPolicies/228353087260/servicePerimeters/terra_dev_aou_test",
+      "authDomainName": "all-of-us-registered-test",
+      "authDomainGroupEmail": "GROUP_all-of-us-registered-test@dev.test.firecloud.org"
+    },
+    "tier2": {
+      "tierLabel": "Prototype",
+      "servicePerimeterName": "accessPolicies/228353087260/servicePerimeters/terra_dev_aou_test_2",
+      "authDomainName": "all-of-us-test-prototype-3",
```

Comment:
Is this necessary for the prototype, or just something you know we'll need soon?

Reply:
Yes - in addition to the Service Perimeter and Folder, the Auth Domain is a key part of how we're currently implementing the Registered Tier.  I want to see that we can work with multiple Auth Domains, one per tier.